ZShare/ViewModels/ExtensionViewModel.swift

```diff
@@ -1183,6 +1183,10 @@ final class ExtensionViewModel {
                 guard let response = responses.first else { return .unknown }
                 return alamoErrorRequiresAbort(.responseValidationFailed(reason: .unacceptableStatusCode(code: response.code)), url: nil, libraryId: libraryId)
 
+            case .authorizationFailed(let statusCode, let response, _):
+                guard statusCode == 413 else { return .unknown }
```

Comment:
Do we want to limit it to 413? Why not just pass status code for all authorization failures?

Reply:
I guess it doesn't hurt. Check https://github.com/zotero/zotero-ios/pull/915/commits/bfb065874c193df2175909eefbf9f7107963eddf